conf/software-vendors.json

```diff
@@ -182,6 +182,24 @@
       "incomeAndDeductions": [
         "self-employment"
       ]
+    },
+    {
+      "name": "IRIS Accountancy Suite",
+      "url": "https://www.iris.co.uk/products/iris-accountancy-suite/",
+      "phone": "03448155555",
+      "email": "https://www.iris.co.uk/",
+      "website": "https://www.iris.co.uk/products/iris-accountancy-suite/",
+      "filters": [
```

Comment:
missing self employment and uk property filter options

Reply:
Updated correct email

Reply:
Included "sole-trader", "uk-property" and "paid-for" in filters